js/app/Visualization/Animation/AnimationManager.js

```diff
@@ -340,11 +340,11 @@ function(Class,
 
       if (rowidx) {
         var animation = self.animations[rowidx[0]];
-        if (animation.data_view) {
+        if (animation && animation.data_view) {
```

Comment:
What I found after a bit of testing is that if you mouseover the map before the workspace is loaded (either with the request in transit, or before actually making the request) the animations won't be there and the animation will be undefined. It prints an annoying error on the console, but it doesn't break anything.